examples/react-map-gl-draw/app.js

```diff
@@ -82,83 +67,53 @@ export default class App extends Component {
     });
   };
 
-  _renderToolbox = () => {
-    const drawModes = MODES.filter(mode => mode.value.startsWith('DRAW'));
-    const otherModes = MODES.filter(mode => !mode.value.startsWith('DRAW'));
+  _switchMode = evt => {
+    let selectedMode = evt.target.id;
+    if (selectedMode === this.state.selectedMode) {
+      selectedMode = null;
+    }
+
+    this.setState({
+      selectedMode,
+      selectedFeatureId: null
+    });
+  };
+
+  _renderToolbar = () => {
     return (
-      <div style={ToolboxStyles.toolbox}>
-        <ToolboxRowWrapping>
-          <ToolboxLabel style={{ paddingLeft: '2px' }}>Modes</ToolboxLabel>
-          <ToolboxRow>
-            {otherModes.map(mode => (
-              <ToolboxButton
-                id={mode.value}
-                key={mode.value}
-                style={{
-                  backgroundColor: this.state.mode === mode.value ? '#a0cde8' : ''
-                }}
-                onClick={this._switchMode}
-              >
-                {mode.name}
-              </ToolboxButton>
-            ))}
-          </ToolboxRow>
-          <ToolboxRow>
-            {drawModes.map(mode => (
-              <ToolboxButton
-                id={mode.value}
-                key={mode.value}
-                style={{
-                  backgroundColor: this.state.mode === mode.value ? '#a0cde8' : ''
-                }}
-                onClick={this._switchMode}
-              >
-                {mode.name}
-              </ToolboxButton>
-            ))}
-          </ToolboxRow>
-        </ToolboxRowWrapping>
-        <ToolboxDivider />
-        <ToolboxRowWrapping>
-          <ToolboxRow>
-            <ToolboxButton onClick={this._onDelete}>Delete</ToolboxButton>
-          </ToolboxRow>
-        </ToolboxRowWrapping>
-      </div>
+      <Toolbar
+        selectedMode={this.state.selectedMode}
+        onSwitchMode={this._switchMode}
+        onDelete={this._onDelete}
+      />
     );
   };
 
-  _switchMode = evt => {
-    const mode = evt.target.id;
-    this.setState({
-      mode,
-      selectedId: null
-    });
+  _getEditHandleShape = ({ feature }) => {
+    return feature.properties.renderType === 'Point' ? 'circle' : 'rect';
   };
 
   render() {
-    const { viewport, mode, selectedId, features } = this.state;
+    const { viewport, selectedMode, selectedFeatureId, features } = this.state;
     return (
       <MapGL
         {...viewport}
         ref={_ => (this._mapRef = _)}
         width="100%"
         height="100%"
-        mapStyle="mapbox://styles/mapbox/light-v9"
+        mapStyle="mapbox://styles/uberdata/cive48w2e001a2imn5mcu2vrs"
```

Comment:
User's won't be able to run this with their tokens. Can we inject our private style using webpack configs?